content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -361,8 +369,30 @@ rules for deprecation are as follows:
 **Rule #7: Deprecated behaviors must function for no less than 1 year after their
 announced deprecation.**
 
-This does not imply that all changes to the system are governed by this policy.
-This applies only to significant, user-visible behaviors which impact the
+If the feature or behavior is being replaced with an alternative implementation
+that requires work to adopt the change, there should be an effort to simplify
+the transition whenever possible. If an alternative implementation is under
+Kubernetes organization control, the following rules apply:
+
+**Rule #8: The feature of behavior must not be deprecated in favor of an alternative
+implementation that is less stable**
+
+For example, GA feature cannot be deprecated in favor of a Beta replacement.
+We should, however, encourage the transition to the alternative implementation
+even before it reached the same maturity level. This is particularly important
```

Comment:
```suggestion
The Kubernetes project does, however, encourage users to adopt and transitions to alternative
implementations even before they reach the same maturity level. This is particularly important
```

We [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).